store/src/java/com/zimbra/cs/imap/ImapRequest.java

```diff
@@ -610,6 +614,51 @@ String readSequence() throws ImapParseException {
         return validateSequence(readContent(SEQUENCE_CHARS), true);
     }
 
+    CacheEntryType[] readCacheEntryTypes() throws IOException, ImapParseException {
+        String[] cacheStrTypes = readAstring().split(",");
```

Comment:
I implemented this change, with the exception that a blank list is not permitted; this is because FlushCacheRequest requires at least one cache type, and I figured this would be consistent.